<launch>
 <!-- Turn on hybrid_astar node -->
            <node name="hybrid_astar" pkg="hybrid_astar" type="hybrid_astar" output="screen"> 
                <param name="simul" type="bool" value="true" />
                <param name="reverse" type="bool" value="true" />
                <param name="voronoibased" type="bool" value="true" />
                <param name="useReedsSheppShot" type="bool" value="false" />
            </node>
            <node name="rviz2" pkg="rviz" type="rviz" args="-d $(find hybrid_astar)/../rviz/standalone.rviz" />
</launch>
